server/data/restClient.ts

```diff
@@ -10,7 +10,7 @@ import { restClientMetricsMiddleware } from './restClientMetricsMiddleware'
 
 interface GetRequest {
   path?: string
-  query?: string
+  query?: object | string
```

Comment:
This change allows us to use numbers as query string parameters, such as the page number (they will get coerced into strings by the time the actual HTTP request is sent over the wire); it's just a convenience for us to be able to use numbers in code 👍